pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -1546,7 +1546,7 @@ preserved.
 ][
     pmd-java:typeIs('java.lang.AutoCloseable')
     or
-    ../../PrimarySuffix/Arguments[@ArgumentCount = 1]//PrimaryPrefix[pmd-java:typeIs('java.lang.AutoCloseable')]
+    ../../PrimarySuffix/Arguments//PrimaryPrefix[pmd-java:typeIs('java.lang.AutoCloseable')]
```

Comment:
this actually checks at least 1 argument is an autoclosable… should we check **all** arguments are autoclosable by doing

```
not(../../PrimarySuffix/Arguments//PrimaryPrefix[not(pmd-java:typeIs('java.lang.AutoCloseable'))])
```

Reply:
@jsotuyod Actually, when I added the test case containing close method with multiple arguments, it was failing when [@ArgumentCount = 1], so I removed it. So, if we want to have a check for at least one argument in the close method, can we do [@ArgumentCount >= 1]? I checked and it is working.

Reply:
Nevermind… if the user configured the close method this is probably good enough